WTWSMS/localisation/WtWSMS_artifacts.csv

```diff
@@ -1,9 +1,11 @@
 #CODE;ENGLISH;FRENCH;GERMAN;;SPANISH;;;;;;;;;x
 western_regalia;Western Roman Imperial Regalia;Regalia de l'Empire romain d'Occident;;;Regalia del Imperio Romano Occidental;;;;;;;;;x
 western_regalia_desc;The Imperial Regalia of Rome are a collection of various objects carried in tow during processions of the emperor. The main items were various lances and spears along with a standard, which was an embroidered banner surmounted by a gold eagle.;La regalia imp�riale de Rome est une collection d'objets divers port�s au cours des processions de l'empereur. Les objets principaux �taient diff�rentes �p�es et lances avec un �tendard, qui �tait une banni�re brod�e surmont�e d'un aigle dor�.;;;La Regalia Imperial de Roma es una colecci�n de varios objetos llevados en remolque durante las procesiones del emperador. Los art�culos principales eran varias lanzas y lanzas junto con un est�ndar, que era una bandera bordada coronada por un �guila del oro.;;;;;;;;;x
-silk_dress;Silk Dress;Robe en soie;;;;;;;;;;;;x
-silk_dress_desc;A silk dress.;Une robe de soie.;;;;;;;;;;;;x
-chess_set;Chess Set;Jeu d'�checs;;;;;;;;;;;;x
-chess_set_desc;An old, worn chess set.;Un jeu d'�checs vieux et us�.;;;;;;;;;;;;x
-world_map;World Map;Carte du monde;;;;;;;;;;;;x
-world_map_desc;A world map.;Une carte du monde;;;;;;;;;;;;x
\ No newline at end of file
+silk_dress;Silk Dress;Robe en soie;;;Vestido de seda;;;;;;;;;x
+silk_dress_desc;A silk dress.;Une robe de soie.;;;Un vestido de seda;;;;;;;;;x
+chess_set;Chess Set;Jeu d'�checs;;;Juego de ajedrez;;;;;;;;;x
+chess_set_desc;An old, worn chess set.;Un jeu d'�checs vieux et us�.;;;Un viejo y da�ado juego de ajedrez;;;;;;;;;x
+world_map;World Map;Carte du monde;;;Mapamundi;;;;;;;;;x
+world_map_desc;A world map.;Une carte du monde;;;Mapamundi;;;;;;;;;x
+gurzil_image;Image of Gurzil;;;;Imagen de Gurzil;;;;;;;;;x
+gurzil_image_desc;A image of Gurzil;;;;Una Imagen de Gurzil;;;;;;;;;x
```

Comment:
Typo here, since "i" is a vocal in English it should be "an".

Reply:
Oops. Solved